defaults/main.yml

```diff
@@ -2,9 +2,10 @@
 # WARNING: This may not be your desired MGMT address.
 nova_my_ip: '{{ ansible_default_ipv4.address }}'
 # RabbitMQ
-rabbit_hostname: 'localhost'
-rabbit_username: 'rabbit_username_default'
-rabbit_password: 'rabbit_pass_default'
+openstack_nova_compute_rabbit_hostname: 'localhost'
+openstack_nova_compute_rabbit_username: 'rabbit_username_default'
+openstack_nova_compute_rabbit_password: 'rabbit_pass_default'
```

Comment:
And in any way, this password needs to be known playbook-wise, so I don't know if it's a good idea to ask for it in the role itself.